_posts/2016-04-14-using-bumbler-to-reduce-runtime-dependencies.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Using Bumbler to reduce runtime dependencies"
+date: 2016-04-14 11:53:00
+categories: ["ruby", "rails", "performance"]
```

Comment:
@mauro-oto I think it'd be more accurate if you label this with `["ruby", "performance"]` as it's not really Rails-specific.